### PR TITLE
Move reaction_schema to dc_bundle

### PR DESCRIPTION
**Pull request by yiqunw700**
Also copy figma_schema::Vector as Vector under dc_bundle.

### CODE REVIEW

File: crates/figma_import/src/reaction_schema.rs [hunk 1]
```diff
@@ -476,44 +64,11 @@ fn parse_reactions() {
...
-
-#[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Copy)]
-#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
-pub enum OverlayBackgroundInteraction {
-    None,
-    CloseOnClickOutside,
-}
-
 #[test]
 fn parse_frame_extras() {
```
Comment (yiqunw700) [inline crates/figma_import/src/reaction_schema.rs:68]: Let me know if the tests should also be moved to dc_bundle.

File: crates/dc_bundle/src/legacy_definition/element/vector.rs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+use serde::{Deserialize, Serialize};
+
+// XXX ColorStop, Transform
```
Comment (timothyfroehlich) [inline crates/dc_bundle/src/legacy_definition/element/vector.rs:18]: Do you happen to know what this comment is supposed to mean?

**Review by timothyfroehlich — COMMENTED** [review]
So we need to do this a bit differently. The "Json" versions of the structs all need to stay in `figma_import`. They're the data that's being fetched from Figma, so that's why they need to stay there. So with that, the conversions from the "Json" versions also need to stay in `figma_import`, so they'll need to be "turned around", sort of, so that they're implementations on the "Json" structures, instead of implementations on the "non-Json" structures.

**Review by timothyfroehlich — CHANGES_REQUESTED** [review]
(Also if you move the Json versions back then it's more clear that the tests should stay in `figma_import`)

File: crates/dc_bundle/src/legacy_definition/element/vector.rs (hunk 2 above)
Comment (yiqunw700) [reply crates/dc_bundle/src/legacy_definition/element/vector.rs:18]: Not really. @rylin8 Robert do you know what does this comment mean?

**Comment by yiqunw700** [comment]
> So we need to do this a bit differently. The "Json" versions of the structs all need to stay in `figma_import`. They're the data that's being fetched from Figma, so that's why they need to stay there. So with that, the conversions from the "Json" versions also need to stay in `figma_import`, so they'll need to be "turned around", sort of, so that they're implementations on the "Json" structures, instead of implementations on the "non-Json" structures.

Done.

File: crates/figma_import/src/reaction_schema.rs (hunk 1 above)
Comment (yiqunw700) [reply crates/figma_import/src/reaction_schema.rs:347]: Done.

File: crates/dc_bundle/src/legacy_definition/element/vector.rs (hunk 2 above)
Comment (rylin8) [reply crates/dc_bundle/src/legacy_definition/element/vector.rs:18]: Not exactly, though usually @iamralpht adds the `// XXX` comments to indicate something we should fix or add support for in the future.

**Review by rylin8 — APPROVED** [review]

**Review by timothyfroehlich — APPROVED** [review]
Looks great!

**Pull request merged**